[2023-04-06T04:44:21.882+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T04:30:00+00:00 [queued]>
[2023-04-06T04:44:22.055+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-06T04:30:00+00:00 [queued]>
[2023-04-06T04:44:22.127+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T04:44:22.157+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-06T04:44:22.185+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T04:44:22.299+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 04:30:00+00:00
[2023-04-06T04:44:22.341+0000] {standard_task_runner.py:55} INFO - Started process 2707 to run task
[2023-04-06T04:44:22.355+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-06T04:30:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmpew2w127w']
[2023-04-06T04:44:22.376+0000] {standard_task_runner.py:83} INFO - Job 74: Subtask respond
[2023-04-06T04:44:22.753+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-06T04:30:00+00:00 [running]> on host cfece00ff772
[2023-04-06T04:44:23.100+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T04:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-06T04:30:00+00:00
[2023-04-06T04:44:23.110+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-06T04:44:23.247+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T043000, start_date=20230406T044421, end_date=20230406T044423
[2023-04-06T04:44:23.354+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-06T04:44:23.774+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
